pipeline_steps/common.groovy

```diff
@@ -549,6 +549,30 @@ void clone_external_repo(String directory, String repo, String ref, String refsp
 }
 
 
+/**
+ * Merge list of pull requests.
+ *
+ * Iterate through the list of pull request IDs and merge each change
+ * on top of the current HEAD. If a merge fails an exception is raised.
+ *
+ * @param directory location of the Git repository
+ * @param pullRequestIDs ordered list of pull request numbers to merge
+ * @return null
+ */
+void merge_pr_chain(String directory='./', List pullRequestIDs=null){
+    println("Attempting to merge the following pull requests onto the base branch: ${pullRequestIDs}.")
```

Comment:
Nit: inconsistent indentation. Also, we're adding a mix of CamelCase and snake_case names here, any reason for that? I'm not suggesting to change, perhaps we can standardise on this later and update once we've got agreement.

Reply:
The repo is not consistent on any of this stuff. I used snake case for the function name to be consistent with the other functions, `pullRequestIDs` is camel case because there is no consistency in the project and that was what I adopted in gate.groovy. I meant to keep the indentation the same but forgot when I added the println/dir bits. If I update the indentation does that work for you?

Reply:
Totally

Reply:
done